app/views/categories/_category.html.slim

```diff
@@ -1,7 +1,9 @@
-li.list-group-item.d-flex.justify-content-between.align-items-center.w-100.mt-4
-  = category.name
-  span.badge.badge-primary.badge-pill
-    = category.amount
-  span.d-flex.justify-content-end
-    = link_to 'Edit', edit_category_path(category), { remote: true, 'data-toggle' => "modal", 'data-target' => '#modal-window', class: 'btn btn-info mx-3' }
-    = link_to 'Delete', category_path(category), method: :delete, data: { confirm: t('.are_you_sure') }, class: 'btn btn-danger delete-item mx-3'
+div.card.m-2.p-2
+  h5.card-title.d-flex.px-5
+    .text-center= category.name
+    span.badge.badge-primary.badge-pill.ml-auto
+      = category.amount
+  div.card-body.mt-3
+    = link_to t('.add_money'), add_money_category_path(category), {remote: true, 'data-toggle':  "modal", 'data-target': '#modal-window', class: 'btn btn-info mx-3' }
+    = link_to t('.edit'), edit_category_path(category), { remote: true, 'data-toggle' => "modal", 'data-target' => '#modal-window', class: 'btn btn-info mx-3' }
```

Comment:
@sinpai Let's use new ruby syntax. `'data-toggle': "modal"` instead of `'data-toggle' => "modal"`, also in other places